### ISSUE TITLE
Request: always wrap when asked explicitly

### ISSUE THREAD

**Issue by vphantom** [stkb/Rewrap#96]
Hello,

Thank you for Rewrap, a crucial feature somehow missing from vscode.  That said, when dealing with Perl POD, I cannot convince Rewrap to do anything with plain text paragraphs.  I tried selecting a paragraph and the usual ALT-Q, but I have to reflow my documentation manually, which is painful.

What technically prevents Rewrap from obeying when I highlight some text I want rewrapped?

Below is a Perl POD example, although my question is actually not language-specific: I think that ALT-Q should always rewrap what's highlighted (or the current paragraph) even if it is of an unrecognized format as seems to be the case here.  Users can always CTRL-Z back if the result isn't satisfactory.

```perl
=head1 DESCRIPTION

=head2 Overview

This is a paragraph which Rewrap will refuse to touch even though it's essentially plain text.

=cut

use strict;
use warnings;

# This is a Perl comment which
# Rewrap will gladly handle.
```

**Comment by dhagrow** [comment]
I had this same question and found this issue. A workaround I've found is to comment my selection (ctrl+/), wrap it (alt+q), then uncomment (ctrl+/). :man_shrugging:

**Comment by tmewett** [comment]
I think this is a really crucial feature, and the lack of it is making me look for other plugins. I often run into certain syntax which this plug-in won't let me wrap, such as Python doc strings. I think this should be an easy tweak for any reader familiar with extension programming?

**Comment by stkb** [comment]
This is actually how the extension worked to begin with, but I had to change it. I don't remember off the top of my head why exactly, but I will look it up

**Comment by fy2** [comment]
@stkb Have you had a chance to look into this? Can we be given the option to simply select a lump of text and use rewrap to wrap it rather than it being recognised as wrappable comment by Rewrap itself?

**Comment by stkb** [comment]
@fy2 Yeah, I don't have the all the details any more but basically it made the code much simpler not to have this.

However since this is a popular request I will add it back. I am working on something of a rewrite anyway and will add it into there.

As another workaround in the mean time you can also copy the text into a new buffer, and it will be wrapped as plain text in there, and then copy it back.

**Comment by bmartin** [comment]
As another work-around, you can change the file mode to plain text and then rewrap.

I ran into this with pod as well. Looking forward to seeing this when available.

**Comment by jeanprey** [comment]
Hello,

I would really appreciate the ability to reformat any text. The ability to reformat just the comments in a selected text is excellent and should be kept. I understand that addressing both features at the same time is difficult. However, what about a separate command that would unconditionally rewrap the selected text, without considering any comment markers, just keeping indentation?
